lib/krikri/aggregation_entity_behavior.rb

```diff
@@ -0,0 +1,14 @@
+
+# TODO:  move to a subdirectory?
```

Comment:
Yes, that seems best.  I was thinking about a general `behaviors` directory, but that would be out of scope here, and would not make all of the other classes easier to find.